homeassistant/components/fan/__init__.py

```diff
@@ -42,6 +42,7 @@
 
 SPEED_OFF = 'off'
 SPEED_LOW = 'low'
+SPEED_MEDIUM_LOW = 'medium-low'
```

Comment:
This requires approval in an issue in our architecture repo.

https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model